### ISSUE TITLE
a11y: overlapping widgets don't block pointer events on iOS

### ISSUE THREAD

**Issue by goderbauer** [flutter/flutter#13454]
Create an overlay over a dismissible barrier. The overlay should have same whitespace that doesn't allow user interaction between elements that do allow user interaction. During touch exploration, when the user now moves the finger into one of those whitespace areas, the a11y focus will move to the underlaying dismissible barrier. This can be very confusing and makes it hard to find the elements in the overlay that the user can interact with.

**Comment by goderbauer** [comment]
It looks like the only way to block pointer events in a11y mode is to use actual UIViews, e.g. if the overlay were an actual UIView, it would not be transparent to hit testing in a11y mode.

There doesn't seem to be an API to limit hit testing between custom accessibility elements within the same View.
